src/adapter/aws-lambda/handler.ts

```diff
@@ -258,12 +278,36 @@ export abstract class EventProcessor<E extends LambdaEvent> {
 
   protected abstract setCookiesToResult(result: APIGatewayProxyResult, cookies: string[]): void
 
+  protected getHeaderValue(headers: E['headers'], key: string): string | undefined {
+    let value = headers?.[key]
```

Comment:
It's a matter of taste, but we often use the ternary operator to reduce the use of `let`. What do you think about writing this?

```ts
    const value = headers
      ? Array.isArray(headers[key])
        ? headers[key][0]
        : headers[key]
      : undefined
```